src/cms/static/js/tinymce_plugins/autolink_phone/index.js

```diff
@@ -0,0 +1,3 @@
+// Entry point for Tinymce hook.
+// Author: Jan-Ulrich Holtgrave (holtgrave@integreat-app.de)
+require('./plugin.js');
```

Comment:
This npm-style syntax normally doesn't work in browsers, right? Or does tinymce perform some magic here?
Or can this file just be omitted, since only `plugin.js` is listed in the `external_plugins`?

Reply:
I guess we might remove this as we are not really publishing the plugin, but Tinymce does recommend that a plugin should include at least index.js, plugin.js and plugin.min.js